lib/_stream_readable.js

```diff
@@ -129,6 +129,11 @@ function ReadableState(options, stream, isDuplex) {
   // Should .destroy() be called after 'end' (and potentially 'finish')
   this.autoDestroy = !!(options && options.autoDestroy);
 
+  // Indicates whether the stream has errored. When true all write() calls
+  // should return false. This is needed since when autoDestroy
+  // is disabled we need a way to tell whether the stream has failed.
+  this.errored = false;
```

Comment:
This comment looks incorrect, as there is no `write` method in `Readable`.

Reply:
sorry, fixed